app/models/book.rb

```diff
@@ -0,0 +1,5 @@
+class Book < ApplicationRecord
+  def name_as_kebab
```

Comment:
We should integrate this into other models as we make them, its a really nice way to format a dynamic id or class for html pages, looks perfect.